docs/get-started/training-certification-faq.md

```diff
@@ -0,0 +1,287 @@
+---
+id: training-certification-faq
+title: Sumo Logic Training and Certification FAQ
+sidebar_label: Training and Certification
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Learning at Sumo Logic will help you build the skills necessary to achieve your organization's goals using our platform to make the world’s apps reliable and secure.
+
+We offer courses online (live and on-demand) and in-person **for free**. Customized training is also available through our private offering.
+
+* [**Sumo Logic Training Portal**](#training-portal), only available in-product, where you'll find our courses and certifications. To browse our certification descriptions, see [Sumo Logic Training and Certifications](https://www.sumologic.com/learn/certifications).
+* [**Sumo Logic Training Lab Environment**](#training-lab-environment), curated and set up to work along with our training curriculum.
+* [**Sumo Logic Partner Training**](https://partnertraining.sumologic.com), dedicated technical training for Sumo Logic partners. To sign up, you'll need an access code, which you can obtain from your account manager.
+
+## Training Portal
+
+### How do I access the Training Portal?
+
+1. First, log in to Sumo Logic.
+1. From the **Home** page, click the **Certification** tab.
+1. Click **Get Certified**. You'll be redirected to the portal page, where you can access all courses and certifications.
+
+<details><summary><strong>Get Certified</strong> link not working? Make sure to verify your email address.</summary>
+
+For security reasons, you must must [verify your email address](#how-do-i-verify-my-email-address) in order to access [third-party services within Sumo Logic](#how-do-i-obtain-third-party-service-access). If you have a working Sumo Logic username and password, then your email address is considered verified.
+
+</details>
+
+![certs](/img/get-started/certifications.gif)
+
+
+### How do I verify my email address?
+
+1. Go to **Help** > **Community**.<br/> ![verify-email](/img/get-started/verify-email.gif)
+1. You will be prompted to verify your email. Click **Send Verification Email**.
+1. Check your email inbox for the verification request.
+1. Open the email and click **Verify Your Email**.
+1. Now that your email is verified, you'll be able access to third-party services.
```

Comment:
@zazocar The instructions in the next section say you need to "activate your email" and "active your account" (not "verify your email") in order to access third-party services. So do you need to perform both steps or just one?